pkg/cmdline/flag.go

```diff
@@ -198,16 +198,15 @@ func (m *flagManager) updateCmdFlagFromEnv(cmd *cobra.Command, precedence int, f
 				continue
 			}
 			if precedence > 0 {
-				deprecationFormat := "DEPRECATED USAGE: Environment Prefix [%s] for environment variable will not be supported in the future, use [%s] instead for [%s],"
```

Comment:
@DrDaveD I have to disagree with this change.  all 3 of the messages below are "deprecated used" due to the fact that the prefix IS NOT `APPTAINER_` ...

Reply:
That's not true.  It's not deprecated to set both APPTAINER_ and SINGULARITY_ copies of the same variable; the deprecated case is when only SINGULARITY_ is set.  This set of messages is consistent with the messages on the *ENV_ variables.

Reply:
I consider the main concerns wrt environment handling is to be clear when an override is occurring and inform users when their cli behavior is determined be deprecated variables. Both cases appear satisfied in this PR.